src/core/config.go

```diff
@@ -381,6 +384,10 @@ type Configuration struct {
 		HTTPWriteable              bool         `help:"If True this plz instance will write content back to the HTTP cache.\nBy default it runs in read-only mode."`
 		HTTPTimeout                cli.Duration `help:"Timeout for operations contacting the HTTP cache, in seconds."`
 		HTTPConcurrentRequestLimit int          `help:"The maximum amount of concurrent requests that can be open. Default 20."`
+		HTTPRetry                  int          `help:"The maximum number of retries before a request will give up, if a request is retryable"`
+		HTTPRetryWaitMin           cli.Duration `help:"The minimum amount of time a retryable request will wait before retrying."`
+		HTTPRetryWaitMax           cli.Duration `help:"The maximum amount of time a retryable request will wait before retrying."`
+		HTTPLinearJitterRetry      bool         `help:"Uses linear backoff with jitter instead of exponential backoff. Minimum and Maximum should be changed if you select this."`
```

Comment:
This seems like quite a few parameters and particularly the linear jitter one is quite specific to the implementation. I'd be tempted to put them to sensible defaults and just expose the number of retries as a parameter for now.

Reply:
Sounds good to me, I'll send a new commit. 👍